src/core_plugins/kibana/public/dashboard/dashboard_constants.js

```diff
@@ -4,6 +4,9 @@ export const DashboardConstants = {
   LANDING_PAGE_PATH: '/dashboards',
   CREATE_NEW_DASHBOARD_URL: '/dashboard',
 };
+export const DEFAULT_PANEL_WIDTH = 6;
```

Comment:
I moved these in here because now `panel_state` is including `ui/chrome` and I don't want to force anyone who wants these two conts to also include that behemoth.